ui/components/app/nfts-detection-notice/nfts-detection-notice.js

```diff
@@ -23,12 +25,10 @@ export default function NftsDetectionNotice() {
       <Dialog type="message" className="nfts-detection-notice__message">
         <Box display={DISPLAY.FLEX}>
           <Box paddingTop={1}>
-            <i
-              style={{
-                fontSize: '1rem',
-                color: 'var(--color-primary-default)',
-              }}
-              className="fa fa-info-circle"
+            <Icon
+              name={ICON_NAMES.INFO}
+              className="info-circle"
+              color={IconColor.primaryDefault}
```

Comment:
![Screenshot 2023-03-06 at 4 19 45 PM](https://user-images.githubusercontent.com/39872794/223089623-691985e8-18f5-4dea-b4ea-d76f7e0e8add.jpg)

Reply:
We could probably replace this whole component with `BannerAlert`?